File: targets/jsts-web/src/vulns/vuln32_eval.ts
import { Router } from "express";
export const router = Router();
router.get("/vuln32/calc", (req, res) => {
  const expr = String(req.query.expr || "");
  // VULN:VULN-32:CWE-95:taint code injection — user input passed to eval()
  res.send("result: " + String(eval(expr)));
});

File: targets/python-web/vulns/vuln32_eval.py
from flask import Blueprint, request
bp = Blueprint("vuln32", __name__)

@bp.get("/vuln32/calc")
def calc():
    expr = request.args.get("expr", "")
    # VULN:VULN-32:CWE-95:taint code injection — user input passed to eval()
    return "result: " + str(eval(expr))
